pallets/phala/src/lib.rs

```diff
@@ -1474,11 +1486,7 @@ impl<T: Config> Module<T> {
 						.into(),
 						effects: vec![Order::DepositAsset {
 							assets: vec![MultiAsset::All],
-							dest: Junction::AccountId32 {
-								network: dest_network,
-								id: Encode::encode(&dest_account).try_into().unwrap(),
-							}
-							.into(),
+							dest: dest,
```

Comment:
```suggestion
							dest,
```

shorthand

Reply:
Change to use this shorthand, merge this PR.